modbus4mqtt/modbus4mqtt.py

```diff
@@ -228,22 +248,38 @@ def loop_forever(self):
             self.poll()
             sleep(self.config['update_rate'])
 
+
 @click.command()
-@click.option('--hostname', default='localhost', help='The hostname or IP address of the MQTT server.', show_default=True)
-@click.option('--port', default=1883, help='The port of the MQTT server.', show_default=True)
-@click.option('--username', default='username', help='The username to authenticate to the MQTT server.', show_default=True)
-@click.option('--password', default='password', help='The password to authenticate to the MQTT server.', show_default=True)
-@click.option('--config', default='./Sungrow_SH5k_20.yaml', help='The YAML config file for your modbus device.', show_default=True)
-@click.option('--mqtt_topic_prefix', default='modbus4mqtt', help='A prefix for published MQTT topics.', show_default=True)
-def main(hostname, port, username, password, config, mqtt_topic_prefix):
+@click.option('--hostname', default='localhost',
+              help='The hostname or IP address of the MQTT server.', show_default=True)
+@click.option('--port', default=1883,
+              help='The port of the MQTT server.', show_default=True)
+@click.option('--username', default='username',
+              help='The username to authenticate to the MQTT server.', show_default=True)
+@click.option('--password', default='password',
+              help='The password to authenticate to the MQTT server.', show_default=True)
+@click.option('--mqtt_topic_prefix', default='modbus4mqtt',
+              help='A prefix for published MQTT topics.', show_default=True)
+@click.option('--config', default='./Sungrow_SH5k_20.yaml',
+              help='The YAML config file for your modbus device.', show_default=True)
+@click.option('--insecure', default=True,
+              help='Do not check that the server certificate hostname matches the remote hostname.', show_default=True)
+@click.option('--cafile', default=None,
+              help='path to a file containing trusted CA certificates to enable encrypted.', show_default=True)
```

Comment:
"The path to a file containing trusted CA certificates to enable encryption." would be better.